docker-compose.env

```diff
@@ -1,6 +1,6 @@
 REDACT_IMAGE="docker.brighter.ai/redact:v5.11.2-275"
 REDACT_GPU_IMAGE="docker.brighter.ai/redact-gpu:2.5.0"
-REDACT_UTILS_IMAGE="docker.brighter.ai/redact-utils:1.0.3"
+REDACT_UTILS_IMAGE="docker.brighter.ai/redact-utils:1.1.0"
```

Comment:
I see the docker image in our container registries but on the actual repo it is still v1.0.3 . I guess a short Release on the Repo would be good. :)

Reply:
done